docs/Typedefs.md

```diff
@@ -237,6 +237,36 @@ More information at [axios Interceptors](https://github.com/axios/axios#intercep
 | type | <code>string</code> |  |
 | updatedAt | <code>string</code> | ISO 8601 Extended Format date/time string |
 
+<a name="ContxtApplicationGrouping"></a>
+
+## ContxtApplicationGrouping : <code>Object</code>
+**Kind**: global typedef  
+**Properties**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| applicationId | <code>string</code> |  |
```

Comment:
This should be a `number`

Reply:
I have no idea why I got in my head this was a UUID